judging-algorithm/templates/judges_example.csv

```diff
@@ -0,0 +1,5 @@
+name,inPerson,company
```

Comment:
Do we need the judge's company?  I think we can get rid of this column - we don't need it for the algorithm

Reply:
I also don't think we need the inPerson column LOL since we discussed manually updating the data for those 2 judges this year 

so we should remove data that we're not gonna be using in the algorihtm.